### PR TITLE
Update README and Dockefile

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@wongma7 Please review :)

File: gluster/block/Dockerfile [hunk 1]
```diff
@@ -18,4 +18,3 @@ MAINTAINER Humble Chirammal <hchiramm@redhat.com>
 
 ADD glusterblock-provisioner /usr/local/bin/glusterblock-provisioner
 
-ENTRYPOINT ["/usr/local/bin/glusterblock-provisioner"]
```
Comment (wongma7) [inline gluster/block/Dockerfile]: why remove it? doesn't hurt to have a default

Comment (humblec) [inline gluster/block/Dockerfile]: Let me think some ore on this. Thanks

**Comment by humblec** [comment]
@wongma7 Please review.

**Comment by wongma7** [comment]
lgtm

**Pull request merged**